lib/osf-components/addon/components/paginated-list/layout/template.hbs

```diff
@@ -0,0 +1,37 @@
+{{#if this.loading}}
+    {{#if this.placeholderCount}}
+        <ul class="list-group m-md">
+            {{#each (range 1 @placeholderCount)}}
```

Comment:
Should be `this.placeholderCount`

Reply:
I knew I never should have trusted the `@`...